pkgs/applications/office/appflowy/default.nix

```
@@ -47,6 +47,9 @@ stdenv.mkDerivation rec {
     # Copy archive contents to the outpout directory
     cp -r ./* $out/opt/
 
+    # Copy icon
+    install -Dm444 data/flutter_assets/assets/images/flowy_logo.svg $out/share/icons/appflowy.svg
```

```suggestion
    install -Dm444 data/flutter_assets/assets/images/flowy_logo.svg $out/share/pixmaps/appflowy.svg
```

```suggestion
    install -Dm444 data/flutter_assets/assets/images/flowy_logo.svg $out/share/icons/hicolor/scalable/apps/appflowy.svg
```

AFAIK /usr/share/icons is for specific icon themes.

---

I didn't know that, thanks.

For reference: https://specifications.freedesktop.org/icon-theme-spec/latest/ar01s03.html